tx/manager.go

```diff
@@ -256,8 +256,8 @@ func (tm *TransactionManager) Count(account accounts.Account, tx *types.Transact
 	return count
 }
 
-func (tm *TransactionManager) Start() {
-	go tm.confirmLoop()
+func (tm *TransactionManager) Start(wg *sync.WaitGroup) {
```

Comment:
`Start()` func just start's the services, we can make `wg` in `tx.NewManager` func, not from `rcm` .

Reply:
Updated at 656e43d